metricbeat/module/kubernetes/event/event.go

```diff
@@ -139,28 +139,28 @@ func generateMapStrFromEvent(eve *kubernetes.Event) common.MapStr {
 	}
 
 	output := common.MapStr{
-		"message": eve.Message,
-		"reason":  eve.Reason,
-		"type":    eve.Type,
-		"count":   eve.Count,
+		"message": eve.GetMessage(),
+		"reason":  eve.GetReason(),
+		"type":    eve.GetType(),
+		"count":   eve.GetCount(),
 		"involved_object": common.MapStr{
-			"api_version":      eve.InvolvedObject.APIVersion,
-			"resource_version": eve.InvolvedObject.ResourceVersion,
-			"name":             eve.InvolvedObject.Name,
-			"kind":             eve.InvolvedObject.Kind,
-			"uid":              eve.InvolvedObject.UID,
+			"api_version":      eve.GetInvolvedObject().GetApiVersion(),
+			"resource_version": eve.GetInvolvedObject().GetResourceVersion(),
+			"name":             eve.GetInvolvedObject().GetName(),
+			"kind":             eve.GetInvolvedObject().GetKind(),
+			"uid":              eve.GetInvolvedObject().GetUid(),
 		},
 		"metadata": eventMeta,
 	}
 
 	tsMap := make(common.MapStr)
 
 	if eve.FirstTimestamp != nil {
-		tsMap["first_occurrence"] = eve.FirstTimestamp.UTC()
+		tsMap["first_occurrence"] = kubernetes.Time(eve.FirstTimestamp)
 	}
 
 	if eve.LastTimestamp != nil {
-		tsMap["last_occurrence"] = eve.LastTimestamp.UTC()
```

Comment:
It seems we remove the conversion to UTC. Are all timestamps still UTC?

Reply:
uhm that's a good point, I believe dates must be in UTC anyway, but I'll put the conversion back in, just in case.